add phone guide and some other changes

Sajo811 opened the pull request:
made changes to rainborg guide but didn't change the tone of the guide

ZedPea commented on guides/mining/Mining-with-Phone.md:
```diff
@@ -0,0 +1,45 @@
+4. Enter the password of the pool. For most, it will be `x`
+5. Specify how many cores you want to use. We recommend using half of how many cores your phone has. So if your phone has 8 cores, enter `4`
+6. Click on the image with the three triangles (`<1<1<1`) and tap the 2nd image from the right on the top row, for TurtleCoin
+7. Make sure `Criptonight-lite support (Like AEON)` is checked.
```
Typo, should be "Cryptonight-lite"

p.s., does this app have variant 1 support?

Sajo811 replied:
Wrote it that way since the app has it written this way. I'm pretty sure, it seemed to work on the app for me, and it has TRTL as an option.

Sajo811 commented:
K I'll respond to each seperately
* Kk
* kk
* I branched it from a older commit and was slightly sleep deprived while doing this lol, fixed it
* fixed that too

SoreGums commented:
also of you're going to change italics from underscores, probably need to do all of them ;)

Sajo811 commented:
where else do they exist?
